pass-bakery-fe/src/helpers/formatCurrency.js

```diff
@@ -0,0 +1,9 @@
+export const formatCurrency = (inCurr) => {
```

Comment:
I would have a method name like formatCurrencyToUSD or something so it's clear while reading in component what this method  does.